apps/server/src/modules/user-import/uc/user-import.uc.spec.ts

```diff
@@ -4,8 +4,8 @@ import { ObjectId } from '@mikro-orm/mongodb';
 import { BadRequestException, ForbiddenException, InternalServerErrorException } from '@nestjs/common';
```

Comment:
Out of scope, yet since we touched this, how about adding a test case for line 183?

If I got it right, following should do

			it('should throw if school is already in inUserMigration', async () => {
				school.inUserMigration = true;
				const result = uc.startSchoolInUserMigration(currentUser.id);
				await expect(result).rejects.toThrowError(BadRequestException);
			});
			it('should throw if school has no officialSchoolNumber ', async () => {
				school.officialSchoolNumber = undefined;
				const result = uc.startSchoolInUserMigration(currentUser.id);
				await expect(result).rejects.toThrowError(BadRequestException);
			});

Reply:
I added these. I decreased the threshold in the coverage by one statement as well,